app/controllers/account/news_controller.rb

```diff
@@ -0,0 +1,78 @@
+class Account::NewsController < Account::AccountController
+  rescue_from ActiveRecord::RecordNotFound, with: :not_found
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+
+  before_action :set_news_by_role
+  before_action :set_post, only: %i[show edit update]
+
+  def index; end
+
+  def show; end
+
+  def new
+    @post = News.new
+  end
+
+  def create
+    @post = current_user.news.build(post_params)
+    @post.osbb_id = current_user.osbb_id
+    authorize @post
+    if @post.save
+      redirect_to account_news_index_path
+      flash[:success] = "News \"#{@post.title}\" with id:#{@post.id} has been succesfully created!"
```

Comment:
to know id isn't necessary for user

Reply:
but only lead can create news, not users. I think it will be helpful information for lead